src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/RiskManagementDecisionAnswers.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity
+
+import io.hypersistence.utils.hibernate.type.json.JsonBinaryType
+import jakarta.persistence.Column
+import jakarta.persistence.DiscriminatorValue
+import jakarta.persistence.Entity
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Size
+import org.hibernate.annotations.Type
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.ResidentialChecksTaskAnswerType.MAKE_A_RISK_MANAGEMENT_DECISION
+
+@Entity
+@DiscriminatorValue(value = "MAKE_A_RISK_MANAGEMENT_DECISION")
+class RiskManagementDecisionTaskAnswers(
+  id: Long = -1L,
+  addressCheckRequest: CurfewAddressCheckRequest,
+  taskCode: String,
+  taskVersion: String,
+  @Type(JsonBinaryType::class)
+  @Column(columnDefinition = "jsonb")
+  val answers: RiskManagementDecisionAnswers,
+) : ResidentialChecksTaskAnswer(
+  id = id,
+  addressCheckRequest = addressCheckRequest,
+  taskCode = taskCode,
+  taskVersion = taskVersion,
+  answerType = MAKE_A_RISK_MANAGEMENT_DECISION,
```

Comment:
do we need both task code and answerType? 
maybe there's some benefit keeping these separate

Reply:
Yeah, I just thought task code may change in the future even though it's the same type of answers. But I guess task code is internal info named by us so it would only change if we changed it not with policy version. And if we did get a new task then that would be a new type. I'll have a look at removing.